scripts/install/installer

```diff
@@ -109,6 +109,16 @@ def get_pnodes(config, versions=None, roles=None):
             pnodes.append(JumpScale7(config, pnode['name'], versions))
     return pnodes
 
+def set_cgroup(node, cgroup, cpu_set, memory):
+    _, out, err = node.core.run('cgget  -r memory.limit_in_bytes %s' % cgroup)
```

Comment:
it would probably be nicer to to file_exists /sys/fs/cgroup/memory/{cgroupname}/